build.gradle.kts

```diff
@@ -51,7 +51,7 @@ dependencies {
     compile(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "4.+")
     compile(group = "edu.wpi.first.wpiutil", name = "wpiutil-java", version = "3.+")
     compile(group = "org.controlsfx", name = "controlsfx", version = "8.40.14")
-    compile(group = "com.google.guava", name = "guava", version = "23.0")
+    compile(group = "com.google.guava", name = "guava", version = "23.1-jre")
```

Comment:
That's a weird version specifier

Reply:
Yea.  They changed it.

Reply:
https://groups.google.com/forum/#!topic/guava-discuss/jN49PDKWzqI/discussion